Graphics/GraphicsEngine/interface/GraphicsTypes.h

```diff
@@ -1318,21 +1319,11 @@ struct EngineCreateInfo
 };
 typedef struct EngineCreateInfo EngineCreateInfo;
 
-
 /// Attributes of the OpenGL-based engine implementation
 struct EngineGLCreateInfo DILIGENT_DERIVE(EngineCreateInfo)
 
-    /// Native window handle
-
-    /// * On Win32 platform, this is a window handle (HWND)
-    /// * On Android platform, this is a pointer to the native window (ANativeWindow*)
-    /// * On Linux, this is the native window (Window)
-    void* pNativeWndHandle DEFAULT_INITIALIZER(nullptr);
-
-#if PLATFORM_LINUX
-    /// For linux platform only, this is the pointer to the display
-    void* pDisplay         DEFAULT_INITIALIZER(nullptr);
-#endif
+	/// Native window wrapper
+	NativeWindowHandle NativeWindow;
```

Comment:
Let's just call it `NativeWindow Window`

Reply:
Should I rename the NativeWindowHandle.h file to NativeWindow.h?

Reply:
Yes please